unity-renderer/Assets/Scripts/MainScripts/DCL/FeatureFlags/FeatureFlagConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+using DCL;
+using UnityEngine;
+
+public class FeatureFlagController : IFeatureFlagController
```

Comment:
Should we rename this _file_ to `FeatureFlagController.cs` as well? (instead of `FeatureFlagConfiguration.cs`)

Reply:
+1

Reply:
Good catch!